quantum/keyboard.c

```diff
@@ -291,6 +298,36 @@ void housekeeping_task(void) {
     housekeeping_task_user();
 }
 
+/** \brief Init tasks previously located in matrix_init_quantum
+ *
+ * TODO: rationalise against keyboard_init and current split role
+ */
+void quantum_init(void) {
```

Comment:
This isn't being called anywhere?

Reply:
Looks like I messed up the cherry pick.... should be better now.